src/auxiliary.c

```diff
@@ -0,0 +1,122 @@
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "auxiliary.h"
+
+
+// Default depot path *relative* to the project directory
+// OBS! If you change the value here, you should also update the default value of
+// `LIBTRIXI_JULIA_DEPOT` in `utils/libtrixi-init-julia` accordingly
```

Comment:
Do we have a similar warning in `utils/libtrixi-init-julia`? If yes, does it need to be updated since the value has been moved to a different file?

Reply:
Sure! Thanks!